<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Transfer_money_window</class>
 <widget class="QDialog" name="Transfer_money_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>501</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
background-color:rgba(211, 211, 211, 0.5);
border-radius: 20px;
}

QLabel{
color: black;
}
QLineEdit{
background-color: white;
border-radius: 6px;
}

QComboBox{
background-color:white;
color: black;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Перевести с</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="From_account_line">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Номер карты</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Банковский счёт</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="To_account_line">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>На</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Номер карты</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Банковский счёт</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сумма</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Money_line">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Cancel_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Transfer_button">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>400</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Перевести</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
